### ISSUE TITLE
'Action' or 'Function' should be called 'Callback' in documentation

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#496]
If the documentation about a parameter that is an `Action` or `Func` names it as "Action" or "Function", then we should issue a warning and propose to use the term "Callback" instead.

Example:
```c#
/// <param name="a">
/// The action to invoke.
/// </param>
public void DoSomething(Action a) { }
```

should become:
```c#
/// <param name="a">
/// The callback to invoke.
/// </param>
public void DoSomething(Action a) { }
```
